test/allEqual.js

```diff
@@ -0,0 +1,29 @@
+import { assert } from 'chai';
+
+import * as RA from '../src/index';
+
```

Comment:
Might be worth adding a test for an array of `[1, 1, '1']` or is that actually testing `equal`?

Reply:
Yeah it's testing equal but we can have explicit test for this to demonstrate usecase for different types. I'll add it.